plugins/classifier.cc

```diff
@@ -63,9 +63,9 @@ DEFINE_STRUCTURE_DESCRIPTION(ClassifierConfig);
 ClassifierConfigDescription::
 ClassifierConfigDescription()
 {
-    addFieldDesc("trainingDataset", &ClassifierConfig::dataset,
-             "Dataset for classifier training",
-             makeInputDatasetDescription());
+    addField("trainingData", &ClassifierConfig::trainingData,
+             "Specification of the data for input to the classifier procedure. "
+             "The select statement does not support groupby and having clauses.");
```

Comment:
We do.  Look here https://github.com/mldbai/mldb/pull/10/files#diff-f481bfb5842e848ea2bde3985e6badddR114